rega-genotype/src/main/java/rega/genotype/PhyloClusterAnalysis.java

```diff
@@ -762,7 +764,7 @@ private double runPuzzle(SequenceAlignment alignment)
     }
 
     private String makeResource(File file, String suffix, boolean keep) throws IOException {
-        if (keep) {
+    	if (keep) {
```

Comment:
Please remove all the changes to the whitespaces that have no effect on this file